Sept 2025 spelling (pt.2) (#7375)
umbraco/UmbracoDocs: 10/umbraco-cms/reference/routing/surface-controllers/README.md

* `~/App_Plugins/{areaname}/Views/{controllername}/`
* `~/App_Plugins/{areaname}/Views/Shared/`

> Since you're only able to place static files in your package's `App_Plugin` folder, it is highly recommended to ensure that the area you use matches your package name. This allows your views to be located easily.

Reviewer: ```suggestion
Since you can only place static files in your package's `App_Plugin` folder, it is highly recommended to use a name that matches your package. This helps ensure your views can be found.
```

> Since you're only able to place static files within your package's `App_Plugin` folder, it's highly recommend to ensure that the area you use is the same as your package name, since that allows your views to be found.

Reviewer: ```suggestion
Since you're only able to place static files in your package's `App_Plugin` folder, it is highly recommended to ensure that the area you use matches your package name. This allows your views to be located easily.
```
